docs/installation.rst

```diff
@@ -13,35 +13,6 @@ Install ``pulp_rpm``
 
 Users should install from **either** PyPI or source.
 
-Install ``createrepo_c`` from source
-************************************
-
-``pulp_rpm`` depends on a Python package named ``createrepo_c``, which is compiled from a C
-library. Unfortunately, this package is currently only available as a Python "source distribution",
-meaning that it must be compiled on your own machine. But, luckily, you won't have to do that yourself!
-Simply install the build dependencies on your machine and the build process itself will happen behind
-the scenes when you install the package.
-
-If you are on Fedora, install the build dependencies with this command:
-
-.. code-block:: bash
-
-   sudo dnf install -y gcc make cmake bzip2-devel expat-devel file-devel glib2-devel libcurl-devel libxml2-devel python3-devel rpm-devel openssl-devel sqlite-devel xz-devel zlib-devel
-
-If you are on Ubuntu or Debian, install the build dependencies with this command:
-
-.. code-block:: bash
-
-   sudo apt install -y gcc make cmake libbz2-dev libexpat1-dev libmagic-dev libglib2.0-dev libcurl4-openssl-dev libxml2-dev libpython3-dev librpm-dev libssl-dev libsqlite3-dev liblzma-dev zlib1g-dev
-
-Once the build deps are installed, you can simply install the ``createrepo_c`` package. It is not yet
```

Comment:
You can, however, remove everything from here on down.

Reply:
Also update Travis to remove the step where it downloads and installs from FedoraPeople.

Reply:
And it needs to be added as a dependency to setup.py

Reply:
Oh that's right; let me fix this.

It's because it's an sdist and not a pre-compiled binary?
Are we doing local compilation because the pre-compiled one can't be compiled on the too-old Python environment?
What was the name of that Python build environment again so I can learn more?

Reply:
All these changes are fixed in next push.